develop/smart-contracts/overview.md

```diff
@@ -1,4 +1,145 @@
 ---
 title: Overview
-description: TODO
----
\ No newline at end of file
+description: Learn how developers can build smart contracts on Polkadot by leveraging either WASM/ink! or EVM contracts across many different parachains.
+---
+
+# Overview
+
+## Introduction
+
+Polkadot is a chain specifically designed to service other chains. As a result, developers cannot deploy smart contract applications directly to Polkadot.
+
+Therefore, developers looking to build smart contract applications in Polkadot need to leverage the smart contract functionality that chains in the Ecosystem offer. Developers can generally choose between two main development paths within the ecosystem. 
+
+On one end, developers can leverage WASM-based smart contracts chains that support [ink!](TODO:update-path){target=\_blank}, a Rust-based smart contract language that acts as an embedded domain-specific language (eSDL) for Rust. This allows developers to benefit from Rust’s syntax, type safety, memory safety, and extensive tooling.
+
+Conversely, developers can deploy EVM smart contracts on Ethereum-compatible chains that Polkadot secures. Due to EVM compatibility, developers can use any smart contract language that compiles to EVM bytecode, like [Solidity](https://soliditylang.org/){target=\_blank} and [Vyper](https://vyperlang.org/){target=\_blank}. Additionally, popular wallets like [MetaMask](https://metamask.io/){target=\_blank} can be connected and developers can leverage tools they are familiar with, like [Remix](https://remix.ethereum.org/){target=\_blank}, [Hardhat](https://hardhat.org/){target=\_blank}, [Foundry](https://getfoundry.sh/){target=\_blank}, among others.
+
+This guide highlights some differences between building smart contract applications or parachains within Polkadot's scalable and interoperable framework. It also explores the different development paths for smart contract developers on Polkadot.
+
+!!!info "Parachain Developer?" 
+    If you are a Parachain developer looking to add smart contract functionality to your chain, please check the following sections for [WASM-based](TODO:update-path){target=\_blank} or [EVM-based contracts](TODO:update-path){target=\_blank}.
+
+## Smart Contracts Versus Parachains
+
+In a smart contract, you create a program that executes specific logic isolated to the chain on which it is being executed. All the logic executed is bound to the same state transition rules given by the underlying virtual machine (VM). Consequently, smart contracts are more streamlined to develop, and programs can easily interact with each other through similar interfaces.
+
+``` mermaid
+flowchart LR
+    A[(Chain State)] --> B[[Virtual Machine]]
+    C[Transaction] --> B
+    D["Program Logic and Storage<br/>(Smart Contract)"] --> B
+    B --> E[(New State)]
+    B --> F[Execution Logs]
```

Comment:
The diagram is a bit confussing, because `Program Logic and Storage
(Smart Contract)` are still part of the chain state. What's the goal of this diagram?